static/js/components/ErrorMessage_test.js

```diff
@@ -144,7 +144,7 @@ describe("ErrorInfo", () => {
     });
 
     describe('profile page', () => {
-      it('shows an error when profile get has errored', () => {
+      it('shows an error when profile get has errored instead of leaving the spinner up', () => {
```

Comment:
oh ok sorry I didn't understand what the test was testing. I'll move it back to `ProfilePage_test` since it's super related.